sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -220,7 +220,28 @@ In addition to the auto-rerecording functionality, using the RecordedTestAttribu
 
 ### Recording
 
-When tests are run in `Record` mode, session records are saved to the project directory automatically in a folder named 'SessionRecords'.
+When tests are run in `Record` mode, session records are saved automatically in a local folder named '.assets', located at the root of this repo. This folder will be created automatically by the Test Framework and should not be committed with other changes. Instead, recordings must be pushed manually to the [Azure SDK Assets](https://github.com/Azure/azure-sdk-assets) repo with the help of the `test-proxy` command line tool.
```

Comment:
Do we by-default ignore .assets folder from showing in git commands? If not, should we do that to prevent accidental commit of these folders via git add * etc.?

Reply:
I see .gitignore having .assets in its list, so we are good here!